css/css-ui/historical/moz-user-modify-01.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>-moz-user-modify should be unsupported in author origin</title>
+<link rel="help" href="https://developer.mozilla.org/en-US/docs/Web/CSS/user-modify">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+#modified {
+    -moz-user-modify: read-write-plaintext-only;
```

Comment:
Gecko doesn't support this value, maybe test with `read-write` instead.
```suggestion
    -moz-user-modify: read-write;
```

Reply:
Good catch, I've addressed this feedback